docs/Installation.md

```diff
@@ -677,14 +683,13 @@ you should be able to use the
 API on localhost port 5000 as below. The command requires that `curl` is installed.
 
 ```sh
-curl -H "Content-Type: application/json" -d '{"jsonrpc":"2.0","method":"version_info","id":"1"}' http://localhost:5000/ && echo
+cd `perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`
+./zmtest zonemaster.net

```

Comment:
I suggest
```
$(perl -MFile::ShareDir -le 'print File::ShareDir::dist_dir("Zonemaster-Backend")'`)/zmtest zonemaster.net
```
instead so that you do not have to jump around in the file system.

Reply:
I implemented this.